src/main/resources/db/seed/02_login_history_loadtest.sql

```
@@ -0,0 +1,59 @@
+-- ============================================================
+
+SET SESSION cte_max_recursion_depth = 2000000;
+
+INSERT IGNORE INTO login_history (
+    login_history_id, user_id, ip_address, user_agent,
+    device_fp, country, city, is_suspicious, created_at
+)
+WITH RECURSIVE seq (n) AS (
+    SELECT 1
+    UNION ALL
+    SELECT n + 1 FROM seq WHERE n < 1000000
```

SQLFluff (4.2.2)

[error] The 'WHERE' keyword should always start a new line.

(LT14)